fair/forcing/aerosols.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+from ..constants import molwt
+
+def regress(emissions,
+           aNOx =  -2.62e-3,
+           aSOx =  -5.66e-3,
+           aBC  =  14.28e-3,
+           aOC  =  -8.90e-3,
+           aNH3 =  -5.89e-3):
+
+    """Calculates aerosol forcing based on a multi-linear regression of
+    aerosol emissions to the aerosol ERF time series in AR5.
+
+    Input:
+        emissions:   anthropogenic emissions database
+    Keywords:
+        aSOx, aNOx, aBC, aOC, aNH3:  radiative efficiency of each aerosol
+                     precursor, W/m2/(Mt/yr)
+    Output:
+        F:           aerosol effective radiative forcing
+    """
+
+    em_SOx   = emissions[:,5]
+    em_NOx   = emissions[:,8]
+    em_BC    = emissions[:,9]
+    em_OC    = emissions[:,10]
+    em_NH3   = emissions[:,11]
+
+    em       = np.array([em_SOx, em_NOx, em_BC, em_OC, em_NH3]).T
+    beta     = np.array([aSOx, aNOx, aBC, aOC, aNH3])
+
+    F = np.sum(beta * em, axis=-1)
+    return F
+
+
+def Stevens(emissions, E_SOx_nat=60, alpha=0.001875, beta=0.634, ref_isSO2=True):
+    """Calculates aerosol forcing based on Stevens (2015) that relates sulphate
+    aerosol forcing to SOx emissions in a logarithmic fashion.
+
+    Input:
+        emissions:   anthropogenic emissions database
+    Keywords:
+        E_SOx_nat:   natural emissions of SOx in Mt/yr
+        alpha:       scaling parameter for ERFari
+        beta:        scaling parameter for ERFaci
+        ref_isSO2:   True if E_SOx_nat is in units of SO2 rather than S.
+    Output:
+        F:           aerosol effective radiative forcing
+    """
+
+    factor = 1
+    if ref_isSO2:
+        factor = molwt.SO2/molwt.S
+    em_SOx = emissions[:,5] * factor
+
```

Comment:
Surely the units flag should be related to the units of the input emissions? As it stands, changing natural units modifies the units of em_SOx without checking the value of alpha which seems very dangerous. 

I think it would also be clearer if you renamed `factor` --> `S_to_SO2_conv`

Reply:
This alternative aerosol forcing relationship is from [Stevens 2015](http://journals.ametsoc.org/doi/abs/10.1175/JCLI-D-14-00656.1) and is not the default treatment in v1.1. In this reference natural emissions are in Mt SO2 per year. However, I have assumed that input files use a MAGICC-style spreadsheet (dangerous to assume I know) in which sulphate emissions are in Mt S per year. So, as a general improvement, I think all emissions species need to have names and units attached to them.

Reply:
Perfect, remind me about this tonight